Makefile

```diff
@@ -125,6 +125,11 @@ integration-tests-image-label-json:
 	make -C integration-tests image-label-json ||\
 		( .openshift-ci/slack/notify-if-needed.sh "image-label-json" $$? )
 
+.PHONY: integration-tests-connscraper
+integration-tests-connscraper:
+	make -C integration-tests connscraper ||\
+		( .openshift-ci/slack/notify-if-needed.sh "connscraper" $$? )
```

Comment:
Obviously this matches the other targets in this file but I think we should really avoid having CI-specific logic in the rest of the codebase - one of the most challenging things about the CI port was how coupled everything is

Reply:
I think this issue will be left to a different PR.